### PR TITLE
build: Move rate-limit to cloudflare

### PR DESCRIPTION
**Pull request by istarkov**
## Description

Move rate-limit to cloudflare

## Steps for reproduction

1. click button
2. expect xyz

## Code Review

- [ ] hi @kof, I need you to do
  - conceptual review (architecture, feature-correctness)
  - detailed review (read every line)
  - test it on preview

## Before requesting a review

- [ ] made a self-review
- [ ] added inline comments where things may be not obvious (the "why", not "what")

## Before merging

- [ ] tested locally and on preview environment (preview dev login: 5de6)
- [ ] updated [test cases](https://github.com/webstudio-is/webstudio/blob/main/apps/builder/docs/test-cases.md) document
- [ ] added tests
- [ ] if any new env variables are added, added them to `.env` file

### CODE REVIEW

File: apps/builder/middleware.js [hunk 1]
```diff
@@ -1,467 +0,0 @@
...
- * ```
- */
-import { Ratelimit } from "@upstash/ratelimit";
-import { ipAddress, next as passthroughRaw } from "@vercel/edge";
-import { createClient } from "@vercel/kv";
-import { Hono } from "hono";
-import { HTTPException } from "hono/http-exception";
-import { env as envHono, getRuntimeKey } from "hono/adapter";
-import { handle } from "hono/vercel";
-import memoize from "memoize-one";
```
Comment (TrySound) [inline apps/builder/middleware.js:35]: remove memoize-one too

Comment (istarkov) [reply apps/builder/middleware.js:35]: жалко ;)

Comment (TrySound) [reply apps/builder/middleware.js:35]: у пчелки

Comment (istarkov) [reply apps/builder/middleware.js:35]: мы знаем где ;-)

**Review by TrySound — APPROVED** [review]

**Pull request merged**